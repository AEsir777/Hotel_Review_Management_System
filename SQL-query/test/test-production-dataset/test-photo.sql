/* photos */
CALL GetPhotoDetails ('"Nk-SJhPlDBkAZvfsADtccA"');
CALL GetPhotoDetails ('"jTFl9Cr7RqZqwjH-CI1ZPw"');
CALL GetPhotoDetails (null);
CALL GetPhotoDetails ('"R32_LaNcLXyS2-dtrcuBGw"');
CALL GetPhotoDetails ('"sr-5EY6bmp4jINhea06MjA"');
-- fail
CALL GetPhotoDetails ('"not exist"');