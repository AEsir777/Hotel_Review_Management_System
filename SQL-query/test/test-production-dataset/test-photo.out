/* photos */
\\ CALL GetPhotoDetails ('"Nk-SJhPlDBkAZvfsADtccA"');
15 row(s) returned
// expected output: 15 row(s) returned

\\ CALL GetPhotoDetails ('"jTFl9Cr7RqZqwjH-CI1ZPw"');
6 row(s) returned
"3UNqjjw-WAnrkHoJZ06pfw"	Now offering fresh baked bagels made on site!	food
"6wuJ-q6_315h_zHjkYtm7g"	Perfect to enjoy on the way to shore :)	food
"9JVH-UxentEkb3ffa8hVDg"	Located in a large shopping center with Giant, McDonald's, Wells Fargo, Tuesday Morning and many other fine businesses.	outside
"e0dD0np3hY3F8LoUtrNoPw"	These cream filled donuts can't be beat!  Cream in every bite guaranteed :)	food
"IYypRuEddGxOVKcZ7ZvWPA"	Salted Caramel Cupcakes will drive your taste buds crazy!	food
"pB2PTerPyizq02pogkqLDA"	Raspberry filled butter cookies dipped in chocolate!  This cookie literally melts in your mouth!	food
// expected output:
"3UNqjjw-WAnrkHoJZ06pfw"	Now offering fresh baked bagels made on site!	food
"6wuJ-q6_315h_zHjkYtm7g"	Perfect to enjoy on the way to shore :)	food
"9JVH-UxentEkb3ffa8hVDg"	Located in a large shopping center with Giant, McDonald's, Wells Fargo, Tuesday Morning and many other fine businesses.	outside
"e0dD0np3hY3F8LoUtrNoPw"	These cream filled donuts can't be beat!  Cream in every bite guaranteed :)	food
"IYypRuEddGxOVKcZ7ZvWPA"	Salted Caramel Cupcakes will drive your taste buds crazy!	food
"pB2PTerPyizq02pogkqLDA"	Raspberry filled butter cookies dipped in chocolate!  This cookie literally melts in your mouth!	food

\\ CALL GetPhotoDetails ('"R32_LaNcLXyS2-dtrcuBGw"');
10 row(s) returned
\\ expected output: 10 row(s) returned

\\ CALL GetPhotoDetails ('"sr-5EY6bmp4jINhea06MjA"');
157 row(s) returned
\\ expected output: 157 row(s) returned

-- fail
\\ CALL GetPhotoDetails ('"not exist"');
0 row(s) returned
\\ expected output: 0 row(s) returned

\\ CALL GetPhotoDetails (null);
0 row(s) returned
\\ expected output: 0 row(s) returned